settings.py

```diff
@@ -1,11 +1,14 @@
 import torch
 from decouple import config
+import os
 
-DEVICE = torch.device(config('DEVICE', default='cuda', cast=str))
+DEVICE = torch.device(config('DEVICE', default='cpu', cast=str))
 BATCH_SIZE = config('BATCH_SIZE', default=64, cast=int)
 LATENT_SIZE = config('LATENT_SIZE', default=16, cast=int)
 LEARNING_RATE = config('LEARNING_RATE', default=2e-4, cast=float)
 SAVE_INTERVAL = config('SAVE_INTERVAL', default=10000, cast=int)
 BOARD_SHAPE = config('BOARD_SHAPE', default=(17, 8, 8), cast=tuple)
 GENERATORS_NUMBER = config('GENERATORS_NUMBER', default=1, cast=int)
-DATASETS = config('DATASETS')
+DATABASE = config('DATABASE', default=r"Driver={Microsoft Access Driver (*.mdb, *.accdb)};DBQ="+os.path.join(os.getcwd(),'test.accdb')+";", cast=str)
```

Comment:
sqlite >>> microsoft access, due to e.g. portability (and it still allows you to store the db in a file)

Reply:
I used this database in Access for testing, I do not know how we will ultimately store the database. You can change or delete this default option later

Reply:
that's fine (and if you together make the decision that Access is cross-platform enough then that's fine too)

Reply:
but I could just replace it with a csv file

Reply:
I replaced the .access database with .db, and it works for me on linux as well